frontend/src/components/pages/ModuleViewer.tsx

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import ModuleViewerComponent from "../module-viewer/ModuleViewer";
+
+const ModuleViewer = () => {
+  return <ModuleViewerComponent editable={false} />;
+};
```

Comment:
If this is the entire component, seems a little unnecessary no? I'd call the other component `ModuleViewer` and get rid of this one.